grpc-api/src/main/proto/geese.proto

```diff
@@ -57,4 +67,11 @@ message GeeseLine {
      * and tenth positions of the line.
      */
     repeated int32 geese_positions = 1;
+
+    /*
+     * Array of geese types. For each goose in the geese_position array,
+     * there should be a corresponding goose type on the same position in geese_types array.
```

Comment:
Why not to have a `Goose` message with position and type?

Reply:
I considered it but decided against it so we can illustrate backward and forward compatibility with minimal changes to code and slides.

Reply:
Valid point, but it makes API a little ugly. 
To demonstrate compatibility we can start with showing `Goose` message with one field `position` in it, and add `type` field later. This also demonstrates importance of designing API properly from the very beginning (lessons like avoid boolean fields and primitive arrays because they make APIs less evolvable)

Reply:
Created a GooseLocator message with position and type (as well as CloudLocator)